.github/workflows/_lint.yml

```diff
@@ -52,7 +58,7 @@ jobs:
         shell: bash
         working-directory: ${{ inputs.working-directory }}
         run: |
-          poetry lock --check
```

Comment:
`poetry lock --check` is marked for deprecation `poetry check --lock` is the recommended alternative